client/js/screens/Browse/styles.js

```diff
@@ -0,0 +1,18 @@
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 18,
+    marginHorizontal: 10,
+  },
+  browseImage: {
+    height: 180,
+    width: '94%',
+    alignItems: 'center',
+    justifyContent: 'center',
+    marginHorizontal: 10,
+    marginTop: 10,
```

Comment:
use flexbox properties, resizeMode, use %s for height, width, margin and padding.